src/js/components/Overview/LastIngestion.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback } from 'react';
+import { Typography, Card, Space } from 'antd';
+import { CalendarOutlined } from '@ant-design/icons';
+import { useTranslation } from 'react-i18next';
+
+import { DEFAULT_TRANSLATION } from '@/constants/configConstants';
+import { useAppSelector } from '@/hooks';
+
+const formatDataType = (dataType: string) => {
+  return dataType ? dataType.charAt(0).toUpperCase() + dataType.slice(1) + 's' : 'Unknown Data Type';
+};
+
+const LastIngestionInfo: React.FC = () => {
+  const { t, i18n } = useTranslation(DEFAULT_TRANSLATION);
+  const lastEndTimesByDataType = useAppSelector((state) => state.ingestionData?.lastEndTimesByDataType) || {};
+
+  const formatDate = useCallback(
+    (dateString: string) => {
+      const date = new Date(dateString);
+      return !isNaN(date.getTime())
+        ? date.toLocaleString(i18n.language, {
+            year: 'numeric',
+            month: 'long',
+            day: '2-digit',
+            hour: '2-digit',
+            minute: '2-digit',
+            hour12: true,
+          })
+        : 'Invalid Date';
+    },
+    [i18n.language]
+  );
+
+  return (
+    <>
+      <Typography.Title level={3}>{t('Latest Data Ingestion')}</Typography.Title>
```

Comment:
maybe make this similar to OverviewSection, with a 0-size vertical space wrapper? that way there wont be inconsistent vertical padding

![image](https://github.com/bento-platform/bento_public/assets/357970/61327059-0733-4984-a87e-107cebb6ec2a)

![image](https://github.com/bento-platform/bento_public/assets/357970/44af8ca1-62e0-4af4-9ea4-2a8c720d5fbb)